database/driver.js

```diff
@@ -210,6 +210,8 @@ function determineDataType(data, override) {
  if (data._sub) return "User"
  if (data.members) return "Group"
  if (data.group) return "Project"
+ if (data.total && data.first && data.last) return "Layer"
```

Comment:
We do not have collections in the DB for Canvas or Layer. This shouldn't be here. In fact, Page and Line shouldn't be here either.